sql/src/main/java/io/crate/jobs/JobContextService.java

```diff
@@ -94,11 +95,7 @@ protected void doClose() throws ElasticsearchException {
     }
 
     public JobExecutionContext getContext(UUID jobId) {
```

Comment:
there is a `getContextOrNull` method just below which does exactly the same as this method would do now